pandas/tests/io/parser/common/test_index.py

```diff
@@ -16,6 +16,7 @@
 import pandas._testing as tm
 
 xfail_pyarrow = pytest.mark.usefixtures("pyarrow_xfail")
+skip_pyarrow = pytest.mark.usefixtures("pyarrow_skip")
```

Comment:
can you add some comments here on when to xfail vs skip

Reply:
Yep, will do